c10/core/TensorImpl.cpp

```diff
@@ -357,6 +357,133 @@ bool TensorImpl::is_contiguous_custom(at::MemoryFormat memory_format) const {
       " do not have is_contiguous");
 }
 
+void TensorImpl::empty_tensor_restride_custom(MemoryFormat memory_format) {
+  TORCH_CHECK(
+      false,
+      "Tensors of type ",
+      tensorimpl_type_name(),
+      " do not have empty_tensor_restride");
+}
+
+void TensorImpl::empty_tensor_restride_default(MemoryFormat memory_format) {
```

Comment:
I was inconsistent with header vs. cpp file for default methods, let me know if we want to keep them in the header for inlining reasons or whatever